src/validations.js

```diff
@@ -0,0 +1,12 @@
+export function validateEmail(email) {
+  if (!email) {
+    return 'validation_messages.error_required_field';
+  } else if (!/^([a-z\d[\]])([\w\-.+\][]*)@([ñ\w.\-\][]+)\.[\w\-\][.]{2,}(\?)?.*$/i.test(email)) {
+    return 'validation_messages.error_invalid_email_address';
+  }
+  return null;
+}
+
+export function validateRequiredField(field) {
+  return !field ? 'validation_messages.error_required_field' : false;
```

Comment:
If I am not wrong, it is not compatible with boolean values, what do you think about: 

```suggestion
  return value !== undefined && value !== null && value !== '';
```


```suggestion
  return !value && value !== false;
```

Reply:
changed